code/modules/paperwork/fax.dm

```diff
@@ -0,0 +1,133 @@
+/obj/machinery/fax
+	name = "Fax Machine"
+	desc = "Bluespace technologies on the application of bureaucracy."
+	icon = 'icons/obj/bureaucracy.dmi'
+	icon_state = "fax"
+	density = TRUE
+	power_channel = AREA_USAGE_EQUIP
+	max_integrity = 300
+	pass_flags = PASSTABLE
+	/// The unique ID by which the fax will build a list of existing faxes.
+	var/fax_id
+	/// The name of the fax displayed in the list. Not necessarily unique to some EMAG jokes.
+	var/fax_name
+	/// A reference to an `/obj/item/paper` inside the copier, if one is inserted. Otherwise null.
+	var/obj/item/paper/paper_contain
+	/// Necessary to hide syndicate faxes from the general list
+	var/syndicate_network = FALSE
+
+/obj/machinery/fax/Initialize(mapload)
+	. = ..()
+	if (!fax_id)
+		fax_id = SSnetworks.assign_random_name()
+	if (!fax_name)
+		fax_name = "Unregistered fax " + fax_id
+
+/obj/machinery/fax/Destroy()
+	QDEL_NULL(paper_contain)
+	return ..()
+
+/obj/machinery/fax/update_icon_state()
+	if(paper_contain)
```

Comment:
Not really a standard or requirement, but making this an overlay seems like a better idea than calling update icon state every time we put a paper inside.

Reply:
> Not really a standard or requirement, but making this an overlay seems like a better idea than calling update icon state every time we put a paper inside.

I would do through layers if there is a demand for more functionality, as there is not much functionality to display at the moment.